.gitignore

```diff
@@ -1,5 +1,3 @@
-.vscode
-
```

Comment:
t: A quick search showed my that its not super uncommen to have this shared, but keep in mind that this might be dangerous if some secrets are placed in those files.